src/chrome/content/rules/am730.com.hk.xml

```diff
@@ -0,0 +1,28 @@
+<!--
+	Non-functional hosts
+		Couldn't resolve host name:
+			 - am730.com.hk
+			 - mailserv.am730.com.hk
+			 - www1.am730.com.hk
+
+		Timeout was reached:
+			 - epaper.am730.com.hk
+
+		Secure connection redirects to plaintext:
+			 - archive.am730.com.hk
```

Comment:
I didn't notice that `http(s)://archive.am730.com.hk/` always redirect to `http://archive.am730.com.hk/home` while the host does have a working HTTPS connection. You are correct about the MCB issues.